pkg/ui/handlers.go

```diff
@@ -68,15 +67,8 @@ func (a *API) uiFiles(w http.ResponseWriter, r *http.Request) {
 	a.fileServer.ServeHTTP(w, r)
 }
 
-func (a *API) getCSP(baseURL, kratosPublicURL string) string {
-	b, _ := url.Parse(baseURL)
-	k, _ := url.Parse(kratosPublicURL)
-	additionalScriptURL := ""
-	if k != nil && b != nil && k.Host != b.Host {
-		// Allowlist the kratos URL to allow the browser needs to fetch the webauthn.js script
-		additionalScriptURL = kratosPublicURL
-	}
-	return fmt.Sprintf("default-src 'self' data: https://assets.ubuntu.com; script-src 'self' %v; style-src 'self'", additionalScriptURL)
+func (a *API) getCSP(baseURL string) string {
+	return fmt.Sprintf("default-src 'self' data: https://assets.ubuntu.com; script-src 'self' %v/.well-known/webauthn.js; style-src 'self'", baseURL)
```

Comment:
To make this work for both the juju and the docker deployments we either have to introduce a proxy in docker (probably best as it would be closer to a production env) OR make this configurable